package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "yarn run build:ts && yarn run gen:locale && yarn run build:extension",
     "gen:locale": "yarn run ts-node scripts/generateLocale.ts",
-    "build:ts": "yarn run clean:ts && rollup -c --failAfterWarnings && sed -i '/setTimeout/d' dist/thirdparty/prismjs.js && sed -i 's/var\\ /let\\ /g' dist/extension.js dist/prefs.js",
+    "build:ts": "yarn run clean:ts; rollup -c --failAfterWarnings && sed -i '/setTimeout/d' dist/thirdparty/prismjs.js && sed -i 's/var\\ /let\\ /g' dist/extension.js dist/prefs.js",
```

Comment:
This is just here to block this from being merged for the moment, please leave this unresolved